src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.tsx

```diff
@@ -143,30 +146,52 @@ const SequenceBox = (props: SequenceBoxProps) => {
       <div>Sequence {sequence.id}</div>
       <div className={styles.sequenceHeader}>{sequenceHeaderLabel}</div>
       <div className={styles.speciesCount}>
-        <span className={styles.againstText}>Against</span>
-        {speciesCount} species
+        <span className={styles.againstText}>Against</span> {speciesCount}{' '}
+        species
       </div>
-      <StatusElement jobs={jobs} />
+      {jobs.length ? (
+        <StatusElement sequenceId={sequence.id} jobs={jobs} />
+      ) : null}
     </div>
   );
 };
 
-const StatusElement = ({ jobs }: { jobs: BlastJob[] }) => {
-  const hasRunningJobs = jobs.some((job) => job.status === 'RUNNING');
-  const hasFailedJobs = jobs.some((job) => job.status === 'FAILURE');
+type StatusElementProps = {
+  sequenceId?: number;
+  jobs: BlastJob[];
+};
+
+export const StatusElement = (props: StatusElementProps) => {
```

Comment:
`StatusElement` -> `JobsStatus`? The "element" part in a component name doesn't tell us anything.